Assets/LeapMotion/Core/Editor/LeapVRServiceProviderEditor.cs

```diff
@@ -23,8 +23,8 @@ public class LeapVRServiceProviderEditor : LeapServiceProviderEditor {
                              decorateAllowManualTimeAlignment);
 
       specifyConditionalDrawing(() => { return serializedObject
-                                                 .FindProperty("_disableTemporalWarping")
-                                                   .boolValue == false; },
+                                                 .FindProperty("_enableTemporalWarping")
```

Comment:
My "No no no" comment refers to this change (and the others in the "I don't like inverse checkboxes" commit)